### ISSUE TITLE
list all topcategories: property : Chemical substances Properties service

### ISSUE THREAD

**Issue by egonw** [enanomapper/API#1]
How can I get a lit of all top categories? Is this fixed to the current four, or should we have a method to retrieve a list of top categories?

**Comment by vedina** [comment]
It's a fixed list, corresponding to the fixed four top categories used by ECHA (and OECD Harmonized templates [Endpoint study records](http://iuclid.eu/index.php?fuseaction=home.format&type=public)).

Somebody suggested during the Jan 2015 webinar perhaps "BIO" category can be added, as not all bioassay are about toxicity.

In general the OECD HT correspond roughly to ISO standards, it will be good to be able to compare how these are defined in ISO, and also if there are any changes in the upcoming new version of OECD HT (Iuclid 6)

**Comment by egonw** [comment]
I think this needs to migrate or complemented with ontology searches... I am trying to figure out a good renm example to get all general cytotoxicity data in data.enm.net...

**Comment by vedina** [comment]
It will be better if we can support all of ontology, ISO , OECD HT and other relevant standads, as  they are relevant for different use cases (regulatory, etc.), and have a list of the questions deemed important for the use cases.

For cytotox you don't need the top categories, there are [summary queries](https://apps.ideaconsult.net/enanomapper/query/study) , you get direct links to the corresponding studies. 

Also free text queries (uses ontology mapping, still under dev) can help , e.g. [cytotoxicity](https://apps.ideaconsult.net/enanomapper/ontobucket?search=cytotoxicity&type=protocol&qe=true)

**Comment by egonw** [comment]
Nina, is there an API alternative to /query/study? I don't see it at http://enanomapper.github.io/API/#!/query

**Comment by egonw** [comment]
BTW, I agree with supporting all ontologies...

**Comment by vedina** [comment]
Yes, the place might be not obvious http://enanomapper.github.io/API/#!/substance/getEndpointSummary  = [https://apps.ideaconsult.net/enanomapper/query/study?media=application%2Fjson](https://apps.ideaconsult.net/enanomapper/query/study?media=application%2Fjson)

We can add / move it to /query indeed.

**Comment by vedina** [comment]
+1 for "all ontologies" . May be start gathering list of _"how to retrieve X"_ questions in the [data tracker](https://github.com/enanomapper/data.enanomapper.net/issues) ?
